openshift_scalability/content/quickstarts/eap/eap64-mysql-pv.json

```diff
@@ -675,7 +675,6 @@
                                         "value": "${JGROUPS_CLUSTER_PASSWORD}"
                                     }
                                 ]
-				}
```

Comment:
This change and the other similar ones in this commit don't seem to be necessary.   When I look at the files on master they do not have this extraneous close brace.